lib/cisco_node_utils/bgp_af.rb

```diff
@@ -500,8 +491,7 @@ def default_maximum_paths_ibgp
 
     # Build an array of all network commands currently on the device
     def networks
-      cmds = config_get('bgp_af', 'network', @get_args)
-      cmds.nil? ? default_networks : cmds.each(&:compact!)
+      config_get('bgp_af', 'network', @get_args).each(&:compact!)
```

Comment:
@mikewiebe you wrote this getter method originally I think - can you comment on `compact!` behavior here?

My goal in this PR was to not change any existing behavior, but if this is a bug I can arrange to fix it.